packages/core/chrome/core-chrome-browser-internal/src/ui/project/header.tsx

```diff
@@ -188,7 +189,13 @@ export const ProjectHeader = ({
       <SkipToMainContent />
 
       <HeaderTopBanner headerBanner$={observables.headerBanner$} />
-      <header data-test-subj="kibanaProjectHeader">
+      <header
+        data-test-subj="kibanaProjectHeader"
+        css={css`
+          /* FIXME use a variable for the app bar height */
+          margin-bottom: ${headerActionMenuMounter.mount ? 48 : 0}px;
```

Comment:
@cee-chen I've recently added this change based on Michael's feedback. Based on what you wrote [here](https://github.com/elastic/kibana/pull/162002/files#r1287305125), I take it there might not be a way to fix this hardcoded `48` with a variable, yet.

Reply:
You can use `euiTheme.size.xxxl` for now if you want to use _a_ variable instead of a static number. Based on [Michael's comment here however](https://github.com/elastic/kibana/pull/162002#discussion_r1287331037), I was under the impression that the app bar was going to be a variable height rather than a fixed height.

If so, you may want to consider using a JS resize observer for offset instead 🤷